### PR TITLE
Introduce Livepeer UI

### PR DESCRIPTION
**Pull request by adamsoffer**
**What does this pull request do? Explain your changes. (required)**
This PR introduces the `livepeer-ui` package, a set of primitive react components built with [theme-ui](https://theme-ui.com).

The goal is for this package to be used by all Livepeer products in an effort to unify our brand and product design around a shared visual language, promote constraint-based design principles, promote consistency in how we use patterns and UI elements, and to ship products faster and more efficiently.

**Specific updates (required)**

This package includes the following:
- An initial set of components (`Button`, `Chip`, `Card`, `List`, `ListItem`, `Spinner`, `Logo`). More to come.
- A theme object adhering to the [system-ui spec](https://system-ui.com/).
- A [storybook](https://storybook.js.org/) for viewing and interacting with components in isolation.

**Does this pull request close any open issues?**
Addresses #459 

**Screenshots (optional):**

**Checklist:**
- [x] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by iameli** [comment]
Looking great! Remind me the difference between `<div sx={...}>` and `<Styled.div sx={...}>`? I think that's the one part I don't really follow so far

**Comment by adamsoffer** [comment]
:)

Sure. `<Styled.div>`, `<Styled.h1>`, etc inherit styles defined inside our theme object. So if we have, for example, the following styles assigned to `h1` in our theme object then `<Styled.h1>` will inherit those styles:

```javascript
{
  fonts: {
    body: 'system-ui, sans-serif',
    heading: 'Georgia, serif',
  },
  styles: {
    h1: {
      fontSize: 32,
      fontFamily: 'heading',
      fontWeight: 'heading',
      color: 'primary',
      mt: 4,
      mb: 2,
    },
  }
}
```

I don't actually define any global style to a `div` in our theme object so I should actually remove any instances of `Styled.div`.

File: packages/livepeer-ui/package.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@livepeer/ui",
+  "version": "2.0.0-alpha.1",
+  "scripts": {
+    "dev": "run-p dev:babel",
+    "dev:babel": "babel --ignore node_modules,src/**/*.stories.tsx --out-dir dist -x '.tsx' --watch .",
```
Comment (iameli) [inline packages/livepeer-ui/package.json:7]: Can we add a `dev:storybook` that runs `start-storybook`? Would be nice to run that app alongside everything else when hacking on it.

(I'm thinkin' let's just rename all the existing `start` scripts to `dev` to line up with Now's conventions but still have a nice `yarn run dev` command that starts everything. But that doesn't need to happen in this PR.)

File: packages/livepeer-ui/package.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    "@types/styled-jsx": "^2.2.8",
+    "@types/theme-ui": "^0.2.2",
+    "awesome-typescript-loader": "^5.2.1",
+    "babel-plugin-inline-react-svg": "^1.1.0",
+    "babel-plugin-styled-system": "^0.0.5",
+    "npm-run-all": "^4.1.5",
+    "react-docgen-typescript-loader": "^3.1.1",
+    "typescript": "^3.5.3"
+  },
+  "license": "ISC",
```
Comment (iameli) [inline packages/livepeer-ui/package.json]: We're MIT-licensed everywhere else, probably no reason to switch that up

**Review by iameli — CHANGES_REQUESTED** [review]
Bumping `explorer` and `player`'s `react-scripts` to `^3.1.2` seems to unbreak the CI build - looks great other than that! Feel free to merge when that's in :shipit:

File: packages/livepeer-ui/package.json (hunk 1 above)
Comment (adamsoffer) [reply packages/livepeer-ui/package.json:7]: Yeah good call.

File: packages/livepeer-ui/package.json (hunk 2 above)
Comment (adamsoffer) [reply packages/livepeer-ui/package.json]: oops! Was unintended, I'll switch to MIT.

**Pull request merged**